login.html

```diff
@@ -11,16 +11,26 @@
     <container>
         <div id="logo">
             <a href="index.html">
-                <img src="./image/logo.png" alt="판다마켓 로고">
-                <img src="./image/typography.png" alt="판다마켓 로고 텍스트">
-            </a>    
+                <div class="logo-image">
+                    <img src="./image/logo.png" alt="판다마켓 로고">
+                </div>
+                <div class="logo-text">    
+                    <img src="./image/typography.png" alt="판다마켓 로고 텍스트">
+                </div>    
+                </a>    
         </div>
         <form>
-            <label for="email">이메일</label>
-            <input type="email" id="email" name="user-email" placeholder="이메일을 입력해주세요">
-            <label for="password">비밀번호</label>
+            <div class="email-field">
+                <label for="email">이메일</label>
+                <input type="email" id="email" name="user-email" placeholder="이메일을 입력해주세요">
+                <div class="error-message"></div>
+            </div>
+            <div class="password-field">
+                <label for="password">비밀번호</label>
             <input type="password" id="password" name="user-password" placeholder="비밀번호를 입력해주세요">
-            <button type="submit" class="login-button buttons">
+            <div class="error-message"></div>
+            </div>
+            <button type="button" class="login-button buttons" onclick="location.href='./items.html'" disabled>
```

Comment:
❗️ 수정요청
로그인 버튼은 해당 form에서 submit 이벤트에 해당하므로 button type은 submit이 적절합니다.

```suggestion
            <button type="submit" class="login-button buttons" onclick="location.href='./items.html'" disabled>
```